2015-12-29  Damyan Ivanov  <dam+gnu@ktnx.net>

	[BZ #19385]
	* locales/bg_BG (t_fmt): Change U002C (comma) to U003A (colon).
	(t_fmt_ampm): Likewise.

--- a/localedata/locales/bg_BG
+++ b/localedata/locales/bg_BG
@@ -185,9 +185,9 @@ LC_TIME
 d_t_fmt    "<U0025><U0078><U0020><U0028><U0025><U0061><U0029><U0020>/
 <U0025><U0058><U0020><U0025><U005A>"
 d_fmt      "<U0025><U0065><U002E><U0025><U006D><U002E><U0025><U0059>"
-t_fmt      "<U0025><U006B><U002C><U0025><U004D><U002C><U0025><U0053>"
+t_fmt      "<U0025><U006B><U003A><U0025><U004D><U003A><U0025><U0053>"
 am_pm      "";""
-t_fmt_ampm "<U0025><U006C><U002C><U0025><U004D><U002C><U0025><U0053>"
+t_fmt_ampm "<U0025><U006C><U003A><U0025><U004D><U003A><U0025><U0053>"
 date_fmt   "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
